frontend/src/views/AccountView.vue

```diff
@@ -16,22 +16,23 @@
           { field: 'Full Name', value: [user.full_name] },
           { field: 'Email', value: [user.email] },
         ]" />
-      <SectionBox
+      <CredentialsBox
         :header="'Credentials'"
-        :content= credentialsContent
-        @click="toggleSecret"
+        :clientId="user.clientId"
+        :clientSecret="secretValue"
         data-test="credentials"
         :key="showSecretValue"
-      />
-      <button @click="generateSecret" type="submit">
-        Generate Secret
-      </button>
+        @display-secret= this.onToggleSecret
+        @generateSecret = this.onGenerateSecret
```

Comment:
GenerateSecret event needs to be consistent with how **@display-secrets** is written with the hyphen.